### ISSUE TITLE
WebRequest Class Exception in MacOSX Platform

### ISSUE THREAD

**Issue by tangxiaodao** [picoe/Eto#1426]
##When I use WebRequest Class it cause errors.
##Issue by tangxiaodao@gmail.com

## Expected Behavior

```
Exception:{System.Configuration.ConfigurationErrorsException: Error Initializing the configuration system. ---> System.Configuration.ConfigurationErrorsException: Unrecognized configuration section <runtime> (/Users/EtoTester/TestEto/TestEto.Desktop/bin/Debug/net461/TestEto.Desktop.app/Contents/MonoBundle/TestEto.Desktop.exe.config line 3)
  at System.Configuration.ConfigInfo.ThrowException (System.String text, System.Xml.XmlReader reader) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/ConfigInfo.cs:77 
  at System.Configuration.SectionGroupInfo.ReadContent (System.Xml.XmlReader reader, System.Configuration.Configuration config, System.Boolean overrideAllowed, System.Boolean root) [0x0011f] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/SectionGroupInfo.cs:330 
  at System.Configuration.SectionGroupInfo.ReadRootData (System.Xml.XmlReader reader, System.Configuration.Configuration config, System.Boolean overrideAllowed) [0x00007] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/SectionGroupInfo.cs:273 
  at System.Configuration.Configuration.ReadConfigFile (System.Xml.XmlReader reader, System.String fileName) [0x000ce] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/Configuration.cs:572 
  at System.Configuration.Configuration.Load () [0x00043] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/Configuration.cs:532 
  at System.Configuration.Configuration.Init (System.Configuration.Internal.IConfigSystem system, System.String configPath, System.Configuration.Configuration parent) [0x0005d] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/Configuration.cs:138 
  at System.Configuration.Configuration..ctor (System.Configuration.InternalConfigurationSystem system, System.String locationSubPath) [0x00056] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-10/external/bockbuild/builds/mono-x64/mcs/class/System.Configuration/System.Configuration/Configuration.cs:96 
......
Exception Message:Error Initializing the configuration system.

```

## Actual Behavior

`Can not Execute WebRequest`

## Steps to Reproduce the Problem

  1. Create a Eto Application Project
  2. Create a CustomControl Project with .NetStandard 2.0 Lib
  3. Add CustomControl reference into Eto Application Project

## Code that Demonstrates the Problem
###CustomControl Project sourcecode
`MControl.cs`

```
using System;
using Eto.Forms;
using Eto.Drawing;
using System.Net;

namespace MyCustomControl
{
    public class MControl : Drawable
    {

        public MControl()
        {

        }

        protected override void OnPaint(PaintEventArgs e)
        {
            base.OnPaint(e);
            DrawLine(e.Graphics);
            DrawRectangle(e.Graphics);
        }

        void DrawLine(Graphics g)
        {
            g.DrawLine(Colors.Blue, 50.0f, 50.0f, 150.0f, 150.0f);
        }

        void DrawRectangle(Graphics g)
        {
            g.DrawRectangle(Colors.Red, new RectangleF(50.0f, 50.0f, 150.0f, 150.0f));
        }

        public void GetHttpContent()
        {
            WebRequest request = WebRequest.Create("https://github.com");
        }

    }
}


```
###Eto Application Project sourcecode

`MainForm.eto.cs`

```
using System;
using Eto.Forms;
using Eto.Drawing;
using MyCustomControl;


namespace TestEto
{
    partial class MainForm : Form
    {
        public MControl mControl = new MControl();
        void InitializeComponent()
        {
            Title = "My Eto Form";
            ClientSize = new Size(400, 350);
            mControl.Width = ClientSize.Width;
            mControl.Height = ClientSize.Height;
            Padding = 10;
            Content = new StackLayout
            {
                Items =
                {

                    mControl
                }
            };

        }
    }
}

```

`MainForm.cs`

```
using System;
using Eto.Forms;
using Eto.Drawing;


namespace TestEto
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
            mControl.GetHttpContent();
            
        }
    }
}

```

## Specifications

  - Version: Eto 2.4.1
  - Platform(s): MacOSX,Eto Applications,.NetStandard 2.0
  - Operating System(s): MacOSX 10.14.1

**Comment by cwensley** [comment]
Hey @tangxiaodao, thanks for submitting the issue and the exception.  It really helps narrow down why things aren't working.

It appears that `WebRequest` requires `System.Configuration`, which is a known problem of Eto.Forms for 2.4.1.  Try updating to 2.5.0.  The beta is in prerelease on nuget, or use the [myget feed](https://www.myget.org/F/eto/api/v3/index.json) for bleeding edge releases.

**Comment by cwensley** [comment]
Closing a this is fixed in 2.5.  Please let me know if it doesn't work for you.
